zebra/rib.h

```diff
@@ -518,6 +521,19 @@ static inline struct nexthop_group *rib_active_nhg(struct route_entry *re)
 		return &(re->nhe->nhg);
 }
 
+/*
+ * Is Route Entry Nexthop Object installed and ready to use?
+ */
+static inline bool rib_nhg_installed(const struct route_entry *re)
+{
+	/* Check the installed flag of the fully resolved Nexthop Object */
+	if (CHECK_FLAG(zebra_nhg_resolve(re->nhe)->flags,
+		       NEXTHOP_GROUP_INSTALLED))
+		return true;
+
+	return false;
```

Comment:
`return CHECK_FLAG(zebra_nhg_resolve(re->nhe)->flags, NEXTHOP_GROUP_INSTALLED))`?

Reply:
hmm, I think that suggestion is hard to follow - can we use something less compressed, like:
```
struct nhg_hash_entry *resolved;

resolved = zebra_nhg_resolve(re->nhe);
return check_flag(resolved->flags, NEXTHOP_GROUP_INSTALLED);
```